chromium_src/ios/web/webui/crw_web_ui_scheme_handler.mm

```diff
@@ -0,0 +1,126 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "base/strings/sys_string_conversions.h"
+#include "net/http/http_response_headers.h"
+#include "services/network/public/mojom/url_response_head.mojom.h"
+
+#define startURLSchemeTask originalStartURLSchemeTask
+
+#include "src/ios/web/webui/crw_web_ui_scheme_handler.mm"
+
+#undef startURLSchemeTask
+
+@interface CRWWebUISchemeHandler (Override)
+- (void)webView:(WKWebView*)webView
+    startURLSchemeTask:(id<WKURLSchemeTask>)urlSchemeTask;
+@end
+
+@implementation CRWWebUISchemeHandler (Override)
+- (void)webView:(WKWebView*)webView
+    startURLSchemeTask:(id<WKURLSchemeTask>)urlSchemeTask {
+  GURL URL = net::GURLWithNSURL(urlSchemeTask.request.URL);
```

Comment:
we should never be copying code from upstream like this, it's extremely fragile and I don't even know what the change is. Copying is much worse than patching